Integrate the new Pollbot API

magopian opened the pull request:
Fixes #41

Natim commented on src/App.js:
```diff
@@ -246,100 +250,63 @@ function Dashboard({statuses, version}: DashboardPropType) {
+          Channel: {(releaseInfo && releaseInfo.channel) || ''}
+        </h2>
+        <div className="dashboard">
+          {Object.keys(checkResults).map(key =>
```
We should make sure to display the checks in the order they are in the endpoint listing them so that we can create a timeline and see were it was stuck in the release process.

Natim commented on src/App.js:
```diff
@@ -58,7 +60,7 @@ class ConnectedApp extends React.Component<{dispatch: Dispatch}, void> {
     this.refreshIntervalId = setInterval(
       () => this.props.dispatch(requestStatus()),
-      5000,
+      60000,
```
Can we do it only if some tests are failing and avoid doing it when everything is green?

Natim replied:
In any case the pollbot nginx server is caching for 30s so running it less than 30s after is useless.

Natim commented:
We don't have the loader of the status anymore.

What could be nice is to:

- have a loader for the page until we ge tthe list of checks
- Then display all the checks with a loader in each of them until we've got the status back.
- Add a check for later in case we have an incomplete/error or a missing status

magopian commented:
Indeed, I saw that when implementing the solution to https://github.com/mozilla/delivery-dashboard/pull/48#pullrequestreview-60910136 (in commit https://github.com/mozilla/delivery-dashboard/pull/48/commits/376663f5d8c9588b28299e384e5225283b0d7bc0).

So we should now have the loader for each check, and I'll had a loader for the page. However, I didn't understand what you meant by

> Add a check for later in case we have an incomplete/error or a missing status

Natim commented:
> Add a check for later in case we have an incomplete/error or a missing status

Indeed it is not very explicit, I was talking about: https://github.com/mozilla/delivery-dashboard/pull/48#discussion_r137268479

magopian commented:
I believe I've addressed your comments, and deployed to github pages for you to test ;)

I haven't had the opportunity to test the auto refresh though.

Natim commented:
![emoticon](https://user-images.githubusercontent.com/229453/30155878-57451586-93be-11e7-9445-71b0f9cc4bde.png)

Apparently the emoticon doesn't appear on my nightly/ubuntu computer.

Natim commented:
So if I understand correctly the code, if a check is missing after 60 seconds, you reload the list of checks and then request all checks again. I guess it could work like that.

Natim commented:
I think it is because we don't load the glyphicon css stylesheet properly. It might be because of the fix in #8
